Source/Relativity.Test.Helpers/Relativity.Test.Helpers.NUnit.Integration/ArtifactHelpers/FieldsHelperIntegrationTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using kCura.Relativity.Client;
+using kCura.Relativity.Client.DTOs;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using Relativity.API;
+using Relativity.Test.Helpers.ArtifactHelpers;
+using Relativity.Test.Helpers.ArtifactHelpers.Interfaces;
+using Relativity.Test.Helpers.ServiceFactory.Extentions;
+using Relativity.Test.Helpers.SharedTestHelpers;
+
+namespace Relativity.Test.Helpers.NUnit.Integration.ArtifactHelpers
+{
+	[TestFixture]
+	public class FieldsHelperIntegrationTests
+	{
+		private IFieldsHelper SuT;
+		private IHttpRequestHelper _httpRequestHelper;
+		private IHelper _testHelper;
+		private IServicesMgr _servicesManager;
+		private IRSAPIClient _client;
+
+		private int _workspaceId;
+		private string _workspaceName = $"IntTest_{Guid.NewGuid()}";
+
+		private string _testFieldName = "Production::Sort Order";
+		private int _productionSortOrderFieldArtifactId;
+		private int _productionSortOrderFieldCount = 1;
+
+		[OneTimeSetUp]
+		public void SetUp()
```

Comment:
I see a need to split this into two methods. SetUp and OneTimeSetUp. 

For most of the case, we should instantiate Sut in a SetUp when we have multiple tests written for the Sut.

Reply:
decided to not implement for this class